src/components/FeatureWalkthrough/FeatureWalkthrough.tsx

```diff
@@ -0,0 +1,203 @@
+import { FunctionComponent, useEffect } from 'react'
```

Comment:
```suggestion
import { FunctionComponent, ReactFragment, useEffect } from 'react'
```

Reply:
I added this back in before catching this comment! Please see: b8af56f